examples/twittermap/web/public/javascripts/mapresultcache/services.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Module to cache map result data. Common services module communicates with
+ * middleware and updates resultcache module.
+ */
+'use strict';
```

Comment:
could you teach me what does this `use strict` mean? thanks!

Reply:
Strict enforces certain JavaScript rules,when certain exceptions JavaScript might not throw them.but in strict mode these errors will be reported.

Reply:
thanks!